src/lib/rest/rest.cpp

```diff
@@ -224,6 +224,13 @@ static int httpHeaderGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, co
   else
     LM_T(LmtHttpUnsupportedHeader, ("'unsupported' HTTP header: '%s', value '%s'", ckey, value));
 
+  /* If servicePath was not provided, then use the default value */
```

Comment:
Not sure if in this case we should add an entry into CHANGES_NEXT_RELEASE. For an user point of view, the change doesn't involves an actual change. Maybe something like this could be useful:

```
- Default service path "/" is set explicitely in the code, which may fix some weird behavours when this header was omitted.
```

What do you think?